app/screens/post_options/post_option.js

```diff
@@ -39,8 +39,14 @@ export default class PostOption extends PureComponent {
         text: PropTypes.string.isRequired,
     };
 
+    handleOnPress = () => {
+        setTimeout(() => {
+            this.props.onPress();
+        }, 250);
```

Comment:
Why add a delay here?

Reply:
oh I'll add a comment, the reason for this is to give time to the tap animation to happen so the user gets some UI feedback